models/company.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const location = require('./location');
+
+const companySchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  website: {
+    type: String,
+    required: false,
+  },
+  location: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: location,
+    required: true,
+  },
+});
+
+module.exports = mongoose.model('company', companySchema);
```

Comment:
Use capital letters, applicable to all other schemas

Reply:
Should I use capitals also when referencing it?

Reply:
yes

Reply:
done